lvsfunc/deblock.py

```diff
@@ -142,11 +149,14 @@ def _eval_db(n: int, f: Sequence[vs.VideoFrame],
     return kernel.resample(debl, format=clip.format, matrix=targ_matrix if not is_rgb else None)
 
 
-def vsdpir(clip: vs.VideoNode, strength: VSDPIR_STRENGTH_TYPE = 25, mode: str = 'deblock',
-           matrix: Matrix | int | None = None, tiles: int | Tuple[int] | None = None,
-           cuda: bool | Literal['trt'] = True, i444: bool = False, kernel: Kernel | str = Catrom(),
-           zones: List[Tuple[Range | List[Range] | None, VSDPIR_STRENGTH_TYPE]] | None = None,
-           **dpir_args: Any) -> vs.VideoNode:
+def vsdpir(
+    clip: vs.VideoNode, strength: VSDPIR_STRENGTH_TYPE = 25, mode: str = 'deblock',
+    matrix: Matrix | int | None = None, tiles: int | Tuple[int, int] | None = None,
+    cuda: bool | Literal['trt'] | None = None, i444: bool = False, kernel: Kernel | str = Catrom(),
+    zones: List[Tuple[Range | List[Range] | None, VSDPIR_STRENGTH_TYPE]] | None = None,
+    tilesize: int | Tuple[int, int] | None = None, overlap: int | Tuple[int, int] | None = None,
```

Comment:
Please add all these to the docstring as well. You can just take them as-is from the DPIR docs ig

Reply:
Done